cypress/integration/registration_validation.spec.js

```diff
@@ -175,22 +175,28 @@ describe('User opens registration form and can see validation errors', () => {
   });
 
   it('The User should be able to see validation errors on contributors tab', () => {
-    cy.get('[data-testid=nav-tabpanel-contributors]').click({ force: true });
+    cy.get('[data-testid=nav-tabpanel-contributors]').click();
     cy.get('p.Mui-error').should('be.visible');
     cy.get('[data-testid=nav-tabpanel-contributors]').within(() => cy.get('[data-testid=error-tab]').should('exist'));
 
     // Add author
-    cy.get('[data-testid=add-Creator]').first().click({ force: true });
-    cy.get('[data-testid=search-field] input').first().click({ force: true }).type('test');
+    cy.get('[data-testid=add-Creator]').first().click();
+    cy.get('[data-testid=contributor-modal]').should('be.visible');
+    cy.get('[data-testid=search-field] input').type('test');
+    cy.get('[data-testid=author-radio-button]').first().click({ force: true });
     cy.get('[data-testid=author-radio-button]').first().click({ force: true });
```

Comment:
skjønner ikke hvorfor, men dette virker ikke om jeg bare prøver å trykke en gang. Alt ser greit ut for meg, men testen blir rød. Samme i blokka under når man legger til veileder

Reply:
dette gjelder visst bare for FF...